.github/workflows/ci.yaml

```diff
@@ -24,15 +24,13 @@ jobs:
 
       - name: Set up Helm
         uses: azure/setup-helm@v1
-        with:
-          version: v3.4.0
```

Comment:
This means we will automatically pick up the latest stable release going forward. Needed this because there were some bug fixes in Helm around the `ne` function.